src/content/docs/en/pages/guides/marketplace/templates/cdn-template.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: How to deploy the XXXXXX template
+description: This template helps you to deploy an application with common caching rules to improve the delivery of your content.
+meta_tags: templates, guides, Azion Marketplace
+namespace: docs_guides_XXXXX
+permalink: /documentation/products/guides/XXXXXXXX/
+---
+
+import Badge from '~/components/Badge.astro';
+
+<Badge variant="accent">
+Preview
+</Badge>
+
+The **XXXXXXX** template contains common caching rules and settings to improve the delivery and performance of static and dynamic content, as well as images, based on the file extension of the requested content. Additionally, it also enables image processing and optimization. This way, you can deliver your content using Azion as a CDN.
+
+By deploying this template, the platform creates a new edge application including all the caching rules and settings. Afterward, you can [link your origin with the Azion application](#adding-a-custom-domain) to start using this set of rules with your content and deliver it on the edge.
+
+---
+
+## Requirements
+
+To use this template, you must activate:
+
+<Badge>
+[Application Acceleration](/en/documentation/products/build/edge-application/application-acceleration/)
+</Badge>
+<br/>
+<Badge>
+[Image Processor](/en/documentation/products/build/edge-application/image-processor/)
+</Badge>
+
+You need to enable these modules via Real-Time Manager (RTM). To do so:
+
+1. Access [Real-Time Manager (RTM)](/en/documentation/products/guides/how-to-access-rtm/) > **Account menu**
+2. Select the **Billing & Subscriptions** option.
+3. Select the **Subscriptions** tab.
+4. Activate the switch for each module.
+
+:::caution[warning]
+If a module is activated, the execution of this template could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information.
+:::
+
+---
+
+## Getting the template
+
+To use this template, access [Real-Time Manager (RTM)](https://manager.azion.com/) and select the **Start with a template** option on the homepage.
+
+This will open a new page where you can select the **XXXXXX** card.
+
+---
+
+## Setting up the template
+
+Once you're on the template's page, click the **Settings** tab to open the configuration form and provide the information to configure your application. 
+
+Fields identified with an asterisk are mandatory.
+
+- **Application Name** *: the name of your edge application on Azion.
+    - Use a unique and easy-to-remember name. If the name has already been used, the platform returns an error message.
+- **Origin Address** *: the address of the hosted origin the edge application should fetch the original content from.
+    - Example: `www.yourwebsite.com`
+- **Custom Host Header**: in some cases, it may be necessary to add host information in the header request to retrieve information from the origin. This variable is used for that purpose.
```

Comment:
```suggestion
- **Custom Host Header**: in some cases, it may be necessary to manipulate the value of the Host header request in the request to retrieve information from the origin.
```